spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/chat/memory/cassandra/CassandraChatMemoryAutoConfiguration.java

```diff
@@ -18,8 +18,8 @@
 
 import com.datastax.oss.driver.api.core.CqlSession;
 
-import org.springframework.ai.chat.memory.CassandraChatMemory;
-import org.springframework.ai.chat.memory.CassandraChatMemoryConfig;
+import org.springframework.ai.chat.memory.cassandra.CassandraChatMemory;
```

Comment:
are all vectorstore modules moving their classes to subpackages ? (i don't see that yet with pgvector.  has there been some discussion/agreement to this somewhere we can link to ?)

Reply:
Yes, we are in the process of moving the vector store classes under that structure - in this case - `org.springframework.ai.chat.memory.cassandra` and `springframework.ai.vectorstore.cassandra`.  The issue was that the vector store classes currently use the same package name in all the different modules - `springframework.ai.vectorstore`.

Reply:
was hoping that there was some link you could provide to where the discussion had been had and landed on that conclusion and action.  but it's ok, i trust.